pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+# Description
+
+<!--- Please include a summary of the changes and the related issue. -->
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## Type of change
+
+**Select Type(s) of change:**
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update
+
+# How Has This Been Tested?
+
+<!--- Please describe the tests that you ran to verify your changes. -->
+<!--- Provide instructions that can be reproduced. -->
+<!--- Please also list any relevant details for your test configuration. -->
+
+
+**Test Configuration**:
+(Ignore if not relevant)
+* Python Version:
+* Golang Version:
+* Operating System:
+* Kubernetes Version:
+
+# Checklist:
+
+- [ ] I have performed a self-review of my code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
```

Comment:
we should add:

- `I have fixed any merge conflicts`

Reply:
Will add